### ISSUE TITLE
Feature List - v0.1

### ISSUE THREAD

**Issue by foxycoderr** [Geometrix-Developers/Geometrix#1]
# Feature List

Developing Version: v1.0

Plan:
- [x] add package architecture
- [x] add support of points
- [x] add support of lines
- [x] grouping of points into triangles
- [x] grouping of points into quadrilaterals
- [x] drawing circles
- [ ] documenting current progress

**Comment by FJSAGS** [comment]
Do we have angles as object?

**Comment by foxycoderr** [comment]
No, we don’t, but we don’t need one I thinkVon meinem iPhone gesendetAm 9/2/23 um 3:25 PM schrieb FJSAGS ***@***.***>:﻿
Do we have angles as object?
